src/sentry/static/sentry/app/views/settings/components/settingsLayout.jsx

```diff
@@ -1,10 +1,9 @@
-import {Box, Flex} from 'grid-emotion';
 import PropTypes from 'prop-types';
 import React from 'react';
 import styled from 'react-emotion';
-
 import space from 'app/styles/space';
-
+import {slideInLeft, fadeIn} from 'app/styles/animations';
+import Button from 'app/components/button';
```

Comment:
bit of a nit, but we try and group our imports in order of:

1. 3rd party imports
2. app imports
3. relative imports

Reply:
Fixed in 2032ada